sdk/digitaltwins/azure-digitaltwins-core/CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 ## 1.2.0-beta.1 (Unreleased)
 
 ### Features Added
+- Added interfaces from `com.azure.core.client.traits` to `DigitalTwinsClientBuilder`.
```

Comment:
It would be nice to call out the introduction of `RetryOptions` in the builder as the rest of the traits are not changing anything in the builder for the customer.